TerrainPhysicsCollider docs. (#929)
o3de/o3de.org: content/docs/user-guide/components/reference/terrain/terrain_physics_collider.md

---

> title: Terrain Physics Collider component

Reviewer: ```suggestion
title: Terrain Physics Collider Component
```

> linktitle: TerrainPhysicsCollider

Reviewer: ```suggestion
linkTitle: Terrain Physics Collider
```

> description: ' Open 3D Engine (O3DE) Terrain Physics Collider reference. '

Reviewer: ```suggestion
description: Open 3D Engine (O3DE) Terrain Physics Collider component reference.
```

weight: 100
---

The **Terrain Physics Collider** component provides terrain data to the physics system in the form of a heightfield and material assignments.  
You can configure the dimensions of the collider by modifying the [Axis Aligned Box Shape Component](/docs/user-guide/components/reference/shape/axis-aligned-box-shape) on the same entity.

> The **Box component** is required for the **Terrain Physics Collider** to operate.

Reviewer: ```suggestion
```

Suggest moving this to a Dependencies heading to match style of other components.

Author: Done

## Usage ##

You can assign which materials are assigned to surfaces, by selecting a surface type in the surface pull down menu, then selecting a physics material type in the material pull down.

## Provider ##

[Terrain Gem](/docs/user-guide/gems/reference/environment/terrain)

> 

Reviewer: ```suggestion

## Dependencies

The **Box component** is required for the **Terrain Physics Collider** to operate.

```

---
title: Terrain Physics Collider component
linktitle: TerrainPhysicsCollider
description: ' Open 3D Engine (O3DE) Terrain Physics Collider reference. '
weight: 100
---

The **Terrain Physics Collider** component provides terrain data to the physics system in the form of a heightfield.  

> The dimensions of the collider can be configured by modifying the [Axis Aligned Box Shape Component](/docs/user-guide/components/reference/shape/axis-aligned-box-shape) on the same entity.

Reviewer 2: ```suggestion
You can configure the dimensions of the collider by modifying the [Axis Aligned Box Shape Component](/docs/user-guide/components/reference/shape/axis-aligned-box-shape) on the same entity.
```

Author: Changed.
